var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -58,12 +58,21 @@ class Gromacs(CMakePackage):
                     'AVX2_128', 'AVX2_256', 'AVX_512', 'AVX_512_KNL',
                     'IBM_QPX', 'Sparc64_HPC_ACE', 'IBM_VMX', 'IBM_VSX',
                     'ARM_NEON', 'ARM_NEON_ASIMD'))
-    variant('rdtscp', default=True, description='Enable RDTSCP instruction usage')
+    variant('rdtscp', default=True,
+            description='Enable RDTSCP instruction usage')
+    variant('subcounters', default=False,
+            description='Enable wallcycle subcounters')
+    variant('own_fftw', default=True, description='Build own FFTW')
+    variant('omp_max_threads', default='auto',
+            description='Max number of OpenMP threads',
+            values=('auto', '32', '64', '128', '256'))
+    variant('static', default=False, description='Prefer static libraries')
```

Comment:
Not sure if that is really a good idea as not all deps are static libs and hence gromacs would pick up system libraries.

Reply:
Also, the variant should be called `prefer_static`, not `static`.

Reply:
@junghans, the name that I gave to the option is not great. But as @citibeth pointed out, this is prefer static libraries. So, for those libraries that have static versions, GROMACS picks the static ones. But for those that do not have it still uses the provided ones from Spack.

Reply:
`prefer_static` is fine here.  It's kind of a weird option -- do people actually use this?  Does it help?

Reply:
Sure rename it to `prefer_static`, but doesn't spack mainly build shared libraries and hence enabling this would maybe lead to picking up outside libraries, e.g. `/usr/lib/llbfftw.a` instead of `spack/<some_long_path>/lib/libfftw.so`.

Reply:
If it would, I agree that’s not ideal.